app.js

```diff
@@ -69,6 +69,40 @@ const library = new Library([]);
 
 const addBtn = document.querySelector('.add__add-btn');
 
+/* global luxon */
+/* eslint no-undef: "error" */
+const { DateTime } = luxon.DateTime;
+
+const displayDate = () => {
+  const dateNowFormat = DateTime.now().toFormat('FF');
+  document.querySelector('.date-now').innerText = dateNowFormat;
+};
```

Comment:
- [ ] The current date is supposed to be displayed below the navbar and you should use `Luxon Library` I can see that you tried to use it below but it is not working now please fix it